src/index.js

```diff
@@ -5,32 +5,48 @@ import 'file?name=index.html!../example/index.html';
 import 'react-toolbox/lib/commons.scss';
 import Root from './root';
 import registry from './lib/registry';
+import { forceTrailingSlash } from './lib/pathHelper';
 import './index.css';
 
 if (process.env.NODE_ENV !== 'production') {
   require('./utils.css'); // eslint-disable-line
 }
 
-// Create mount element dynamically
-const el = document.createElement('div');
-el.id = 'root';
-document.body.appendChild(el);
+const fixUrl = (pathname) => {
```

Comment:
I agree with your mention of placement - it would be good to put this somewhere route specific, and try to involve React Router if possible. I like [this approach](https://github.com/ReactTraining/react-router/issues/820#issuecomment-256814655). We could then put the rewriting functions in a separate file under `src/routing`. What do you think?